.github/workflows/build.yml

```diff
@@ -0,0 +1,63 @@
+name: Build
+on: [push]
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      # Cache
+      - name: Cache .gradle/caches
```

Comment:
You also need to cache `testfixtures/offline-repo/build/repo/`

Reply:
I'm not sure that the cache is working properly because the task `:testfixtures-offline-repo:buildOfflineRepositories` is still taking a few minutes...